recipes/r-harmony/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set version = '1.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-harmony
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/harmony_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/harmony/harmony_{{ version }}.tar.gz
+  sha256: a63c7d7cbbc5d183e8f919552e9d73044e0a89660856e80861a00eb5d25ac7b5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - libblas
+    - r-base
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppprogress
+    - r-rhpcblasctl
+    - r-cowplot
+    - r-dplyr
+    - r-ggplot2
+    - r-rlang
+    - r-tibble
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppprogress
+    - r-rhpcblasctl
+    - r-cowplot
+    - r-dplyr
+    - r-ggplot2
+    - r-rlang
+    - r-tibble
+
+test:
+  commands:
+    - $R -e "library('harmony')"           # [not win]
+    - "\"%R%\" -e \"library('harmony')\""  # [win]
+
+about:
+  home: software.broadinstitute.org/harmony
+  license: GPL-3.0-or-later
```

Link is dead; `GPL-3` in DESCRIPTION entails `GPL-3.0-only`:

```suggestion
  home: https://portals.broadinstitute.org/harmony/
  dev_url: https://github.com/immunogenomics/harmony
  license: GPL-3.0-only
```

---

Thanks much for the extra help